web/js/jquery.jmri.js

```diff
@@ -686,6 +687,7 @@
                     jmri.socket.send("time", {});
                 } else {
                     $.getJSON(jmri.url + "time", function (json) {
+                    	if ($.isArray(json)) json=json[0]; //unwrap array                   	
                         jmri.time(json.data.time, json.data);
```

Comment:
The indentation differs for these two lines. It would be good to fix it before merge of this PR. It makes the code easier to read.